search-app/src/cmr/search/site/data.clj

```diff
@@ -55,16 +55,16 @@
   request context which contains the state of a running CMR."
   (fn [context & args] (:execution-context context)))
 
-(defmethod collection-data :cli
-  [context tag provider-id]
-  (as-> (config/application-public-root-url :search) data
-        (format "%scollections" data)
-        (util/endpoint-get data {:accept mt/umm-json-results
-                                 :query-params {:provider provider-id
-                                                :tag-key tag
-                                                :page-size 2000}})
-        (:items data)
-        (sort-by #(get-in % [:umm :EntryTitle]) data)))
+;; (defmethod collection-data :cli
+;;   [context tag provider-id]
+;;   (as-> (config/application-public-root-url :search) data
+;;         (format "%scollections" data)
+;;         (util/endpoint-get data {:accept mt/umm-json-results
+;;                                  :query-params {:provider provider-id
+;;                                                 :tag-key tag
+;;                                                 :page-size 2000}})
+;;         (:items data)
+;;         (sort-by #(get-in % [:umm :EntryTitle]) data)))
```

Comment:
This commented out code should be deleted.

Reply:
See last comment :-)